src/config/consts.js

```diff
@@ -99,7 +99,7 @@ export const consts = {
 
   DIMENSIONS: {
     // the max screen size we want to render full screen
-    FULL_SCREEN_MAX_WIDTH: 428
+    FULL_SCREEN_MAX_WIDTH: 430
```

Comment:
Are there devices with bigger screens? This update is only available for the Pro Max. But have you checked for larger screen devices on the Android platform?

Reply:
you're right, there are plenty devices which require a bigger viewport than the iPhone 14 Pro Max
[Here](https://yesviz.com/viewport/) is a list of viewports for other devices

Reply:
did you check some landscape screens? can you share some screenshots of them as well please

Reply:
yes I also checked them, there are no changes cause 504 as viewport width is still smaller than pretty much all up to date smartphones are in landscape mode 
|iPhone 14 Pro Max) |iPhone SE|
| ------------- | ------------- |
| ![Simulator Screenshot - iPhone 14 Pro - 2023-09-11 at 11 41 25](https://github.com/smart-village-solutions/smart-village-app-app/assets/105894697/3aaebd17-49dc-45c6-b68c-48f9e22d76f9) | ![Simulator Screenshot - iPhone SE (3rd generation) - 2023-09-11 at 11 49 26](https://github.com/smart-village-solutions/smart-village-app-app/assets/105894697/253218a6-3bcf-4814-9ea6-0f516d73b068) |

Reply:
the tab navigation would be interesting in comparison, like in the portait screenshots. can you share them as well please.